### ISSUE TITLE
Design Pass for DPUL Success Dashboard

### ISSUE THREAD

**Issue by sdellis** [pulibrary/dpul-collections#33]
Blocked by #14

##Acceptance Criteria
Wireframe a dashboard page that allows users to determine how successful we are based on a number of metrics that we are getting from Plausible. 

Users should be able to determine success over a given date range for the following:

- Traffic (Basic Site traffic metrics from Plausible)
- Engagement (Aggregate totals for Record Page Views, UV Clicks, and Downloads)
- Sources (Pie graph of top referrers for the given date range)

For each of the above metrics, we should allow for a drill down on data. For example:

- Traffic can default to "Visitors" but allow a user to decide which metric they want to view on the line chart. 
- Engagement would be a tally of "engagement metrics" but one could drill down on the individual data in a line chart.
- Sources could show the top 10 in the pie chart, but could allow for a longer list of all the data. 

We can also supply a table of data at the bottom of the page. The graphs should be given priority at the top of the page. 

## Success Criteria

- [ ] Users should be able to determine how well we did towards traffic and engagement goals for a given time period.

**Comment by AminEdZare** [comment]
Mockup #1 for Success Dashboard

<img width="541" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdy9BQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8292f9350735df3f933fa7ea5d1a8d16b64aa59e/DPUL-Dashboard-Mockup.png" alt="DPUL-Dashboard-Mockup.png" />

**Comment by AminEdZare** [comment]
Mockup #2 for Success Dashboard

- After getting feedback from Trey, we changed the Source section of the mockup to display a line chart instead of a bar graph.
- The "Source Diversity" section was also modified to reflect this change, being renamed "Sources" and displaying an aggregate number instead of a percentage.


<img width="541" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM0xBQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eb16c74255566c9cc9762469d7550615ea9e22e8/DPUL-Dashboard-2.png" alt="DPUL-Dashboard-2.png" />

**Comment by tpendragon** [comment]
I don't totally understand what the 11% and 20% aggregates would tell me, and since they're so big I think I'd give them a lot of mental weight. I wonder if instead of trying to combine those metrics we had their own separate rows, with a little up or down arrow and red/green that indicated "Up 10%" or what have you.

Like if Downloads goes up 30, but Viewer Clicks goes down 100, but RPV goes up 3000, I don't think those things are all connected to the same goals, so I'd want to reflect on those individually if that makes sense? Otherwise I'd see a big green box that was like UP A MILLION PERCENT and be like job done :+1:, but our goal of enabling transformative use may have gone down.

**Comment by tpendragon** [comment]
Also I think "Visitors" isn't in https://github.com/pulibrary/dpul-collections/issues/14, so I wonder if we can take it out. I know it's like...the easy one, but we heard from our stakeholders that it's not really popularity that means success.

**Comment by tpendragon** [comment]
![DPUL-Dashboard-2](https://github.com/user-attachments/assets/8820c9fd-9018-4ab2-9ca7-c7e7219c9ddd)
putting this here to a public link

**Comment by hackartisan** [comment]
I think I was out when we looked at this but since it came up again, reading Trey's comments makes me think it would be nice to have our goals spelled out on this page. Like, what if there was descriptive text that helped link the visuals to our goals and guided someone in how to read / use the info?

**Comment by sdellis** [comment]
Here are some goals: 

-     Number of downloads over a time period goes up. 
-     We want all downloads of Figgy resources, and be able to filter where people are getting to those downloads from. Generally we want downloads to come up.
-     Number of clicks in a session goes up.
-     Number of sessions greater than 1 minute goes up.
-     Users link our data out to other sites and platforms; blogs, IIIF story builders 
-     Both visits from individual sources goes up and diversity of sources goes up
-     Bounce Rate goes down
-     Reference / area librarians or curators (sorry don’t know all the titles) hear that someone used a resource, or help them find a resource they say will be useful.

**Comment by sdellis** [comment]
@AminEdZare and I were talking more about how to visualize these numbers:

For each of the metrics we intend to display a line graph (or scatterplot) detailing daily stats. We will also overlay a "line of best fit" (aka, linear regression) and then use the line's slope to determine the percent of increase or decrease over the time period given. That line of best fit is the indicator for general success or failure over the time period, and the daily values can help determine changes based on events (i.e., publicity campaigns, feature releases, etc.). The Rate of Change percentage would be a good indicator as to the rate of success or failure over a given time period. Rate of Change is calculated as change in Y (metric) / change in X (days). 

For each metric, we will define the metric, state our goal, and indicate how success is determined above the graph. For example we want downloads to go up over time, whereas we want bounce rates to go down (or even what a bounce rate is).
